app/_locales/es_419/messages.json

```diff
@@ -57,118 +72,175 @@
     "message": "Agregar tokens"
   },
   "advanced": {
-    "message": "Avanzada"
+    "message": "Avanzado"
   },
   "advancedOptions": {
     "message": "Opciones avanzadas"
   },
   "advancedSettingsDescription": {
-    "message": "Accede a las funciones de desarrollador, descarga los registros de estado, restablece la cuenta, y configura las redes Testnet y el RPC personalizado"
+    "message": "Acceder a recursos para desarrolladores, descargar registros de estado, restablecer cuenta, configurar redes de prueba y RPC personalizada"
+  },
+  "affirmAgree": {
+    "message": "Acepto"
+  },
+  "aggregatorFeeCost": {
+    "message": "Cuota de red de agregador"
+  },
+  "alertDisableTooltip": {
+    "message": "Esto se puede modificar en \"Configuración > Alertas\""
+  },
+  "alertSettingsUnconnectedAccount": {
+    "message": "Explorando un sitio web con una cuenta no conectada seleccionada"
+  },
+  "alertSettingsUnconnectedAccountDescription": {
+    "message": "Esta alerta aparece en la ventana emergente cuando explora un sitio conectado de Web3, pero la cuenta seleccionada actualmente no está conectada."
+  },
+  "alerts": {
+    "message": "Alertas"
+  },
+  "alertsSettingsDescription": {
+    "message": "Habilitar o deshabilitar cada alerta"
+  },
+  "allowExternalExtensionTo": {
+    "message": "Permitir que esta extensión externa haga lo siguiente:"
+  },
+  "allowOriginSpendToken": {
+    "message": "¿Permitir que $1 gaste su $2?"
+  },
+  "allowThisSiteTo": {
+    "message": "Permitir que este sitio haga lo siguiente:"
+  },
+  "allowWithdrawAndSpend": {
+    "message": "Permitir que se retire $1 y gastar hasta el siguiente importe:"
   },
   "amount": {
-    "message": "Monto"
+    "message": "Importe"
+  },
+  "amountInEth": {
+    "message": "$1 ETH"
+  },
+  "amountWithColon": {
+    "message": "Importe:"
   },
   "appDescription": {
-    "message": "Una billetera de Ethereum en tu navegador",
-    "description": "The description of the application"
+    "message": "Una cartera de Ethereum en el explorador"
```

Comment:
We are changing from 'billetera' to 'cartera' throughout this translation. That's a significant terminological change; I see no problem with comprehensibility, but others may have an opinion.